webserver/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
```

Comment:
Might make sense to put structs into a separate file and import them.

Reply:
Moved them to separate files. No need to explicitly import them, they're in the same package